src/withStyles.jsx

```diff
@@ -99,19 +99,23 @@ export function withStyles(
   function makeCreateFn(direction, stylesInterface) {
     const directionSelector = direction === DIRECTIONS.RTL ? 'RTL' : 'LTR';
     let create = stylesInterface[`create${directionSelector}`] || stylesInterface.create;
+    const original = create;
     if (process.env.NODE_ENV !== 'production') {
       create = withPerf('create')(create);
     }
+    create.original = original;
```

Comment:
mutating the object here can have a performance cost as well; has that been benchmarked in all engines?

It might be better to have this return both functions (original and new) rather than trying to smoosh them together into one value

Reply:
Updated to return both original and wrapped functions without mutating objects.